pkg/tenant/db/projects.go

```diff
@@ -25,7 +27,15 @@ const (
 	ProjectStatusArchived   = "Archived"
 )
 
+type Owner struct {
+	ID       ulid.ULID `msgpack:"owner_id"`
+	Name     string    `msgpack:"owner_name"`
+	Email    string    `msgpack:"owner_email"`
+	Gravatar string    `msgpack:"owner_gravatar"`
```

Comment:
Is the `owner_` prefix necessary for `msgpack`? 

Is the intent to store this inline with the nosql document rather than as a reference to the `members` namespace in order to optimize project list fetches? If so, is there a plan with keeping Owner name/email/gravatar up to date with the Member ID? E.g. we can guarantee that the owner is a member and if member changes, owner will change?

Also Gravatar can be computed from email and doesn't need to be stored in the database.